### PR TITLE
[security] Bump nyc from 13.3.0 to 14.0.0

### PR DESCRIPTION
**Pull request by paulmelnikow**
CI will pass after #3322 is merged.

### CODE REVIEW

**Comment by calebcartwright** [comment]
I've been having some issues with v14 in some of my other projects where coverage isn't being reported correctly on Windows. I believe there's an open issue in the nyc repo along the same lines

Just sharing for awareness in case anyone else sees it in the short term.

**Comment by paulmelnikow** [comment]
The changelog mentions a change in instrumentation include/excludes. I wonder if that could be the problem? Though not sure why it would only take effect on Windows.

Should we hold off updating?

**Comment by calebcartwright** [comment]
> Should we hold off updating?

I'm going to check out this branch on Windows and check. If it looks there's a coverage problem on Win then I'd be inclined to hold off on this. [This is the issue in the nyc repo](https://github.com/istanbuljs/nyc/issues/1065#issuecomment-483825556) I was thinking of, and it's the same type of thing I've been seeing as well

**Comment by calebcartwright** [comment]
Coverage report definitely looks off to me on Windows (report from running all our coverage scripts)

image

**Comment by paulmelnikow** [comment]
It looks like it is not ignoring `.next` or `.github`. Why is `.github` even coming up?

It's not clear it's the same issue as istanbuljs/nyc#1065 though. Hmm…

**Comment by calebcartwright** [comment]
> It's not clear it's the same issue as istanbuljs/nyc#1065 though

I don't know about the OP's issue, but the simple example demonstrated in the last comment is something I'm seeing in a couple private projects. Coverage is fine on folks' Macs but not on Windows

**Comment by calebcartwright** [comment]
I updated my local config to explicitly exclude those directories, and coverage looks better now.

**Comment by calebcartwright** [comment]
My sense is that there's probably just a need to make some minor config tweaks with v14. I pushed the updates I made to my local config that fixed our coverage on Windows, so I believe this is good to merge

File: .nycrc.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
     "test-fixtures",
     "scripts",
     "coverage",
-    "build"
+    "build",
+    ".github",
+    ".next"
```
Comment (paulmelnikow) [inline .nycrc.json]: Would it make sense to delete `.next` on your machine instead? It's not going to be there in recent clones.

Comment (calebcartwright) [reply .nycrc.json]: Ha! Good point

**Pull request merged**